src/RedisEnterpriseCache/examples/Invoke-AzRedisEnterpriseCacheForceDatabaseUnlink.md

```diff
@@ -0,0 +1,6 @@
+### Example 1: Unlink database
+```powershell
+Invoke-AzRedisEnterpriseCacheForceDatabaseUnlink -ResourceGroupName "MyGroup" -ClusterName "MyCache3" -Id @("<database-id>")
+```
```

Comment:
Could you please add sample output with following pattern?
\```output
{code}
\```

Reply:
The given example doesn't have an output, so didn't include it here